### PR TITLE
Update OnAppRegistered, SetAppIcon index

### PR DESCRIPTION
**Pull request by KhrystynaDubovyk**
**OnAppRegistered** index - add p.7, 8 to  MUST section  
**SetAppIcon** index - MUST section was added

### CODE REVIEW

File: docs/UI/SetAppIcon/index.md [hunk 1]
```diff
@@ -10,7 +10,14 @@ Purpose
 : Display the application's specified icon in the HMI's application list.
 
 ### Request
+!!! MUST  
 
+1. Keep track and store the the name and file referencing the app icon each time a successful `SetAppIcon` request occurs from the mobile app.  
+2. Display the requested icon together with the application name in the HMI list of registered applications:
+- Right away if the the list of registered applications is currently displayed
```
Comment (jacobkeeler) [inline docs/UI/SetAppIcon/index.md]: Not formatting correctly, you need to indent new levels of a list with 4 spaces IIRC.

File: docs/BasicCommunication/OnAppRegistered/index.md [hunk 2]
```diff
@@ -48,7 +48,9 @@ If the application does NOT resume data successfully:
   3. Compile and store `VRGrammars` for the `vrSynonyms` parameter, and arrange them for the user to be able to use via voice recognition. Note: The VR commands to activate an application must be accessible when viewing a different active application or the list of registered applications.
   4. Provide the user with the possibility to choose an application among a list of registered applications.
   5. Send an `OnAppActivated` notification to SDL when the user activates an app via the `UI` or `VR`.   
-  6. Manage application events by priority. HMI gets proirity information from _OnAppRegistered_, _UpdateAppList_, _ActivateApp_ HMI API.
+  6. Manage application events by priority. HMI gets proirity information from _OnAppRegistered_, _UpdateAppList_, _ActivateApp_ HMI API.  
+  7. HMI must set app icon and create the app title in the mobile apps list in case SDL provides `<icon>` via `OnAppRegistered` notification.  
+  8. HMI must set default AppLink app icon in case SDL omits `<icon>` at `OnAppRegistered` notification.
```
Comment (jacobkeeler) [inline docs/BasicCommunication/OnAppRegistered/index.md]: Avoid `AppLink` language, this term can probably just be omitted here.

Comment (KhrystynaDubovyk) [reply docs/BasicCommunication/OnAppRegistered/index.md]: fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/95/commits/a83706c3263d276107be618b589895dce7caf8f3

File: docs/UI/SetAppIcon/index.md (hunk 1 above)
Comment (KhrystynaDubovyk) [reply docs/UI/SetAppIcon/index.md]: @jacobkeeler, do you mean new levels of nested list?
tried to fix https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/95/commits/83724107cfd1072bb31e6ded9d165593248b05c6

File: docs/UI/SetAppIcon/index.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ SetAppIcon
 ```json
 {
   "id" : 88,
-  "jsonrpc" : "2.0",
+  "jsonrpc" : "2.
```
Comment (jacobkeeler) [inline docs/UI/SetAppIcon/index.md]: Mistaken deletion, but it breaks the JSON formatting

Comment (KhrystynaDubovyk) [reply docs/UI/SetAppIcon/index.md]: @jacobkeeler , thank you.
fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/95/commits/4b39d1b8620a2725a598945b2dee593e924f6081

**Pull request merged**